tests/test_data/projects/glob_patterns/snowflake.yml

```diff
@@ -0,0 +1,25 @@
+definition_version: 2
+entities:
+  hello_procedure:
+    artifacts:
+      - src/*.*
+    handler: hello
+    identifier:
+      name: hello_function
+    returns: string
+    signature:
+      - name: "name"
+        type: "string"
+    stage: dev_deployment
+    type: procedure
+  my_streamlit:
+    type: "streamlit"
+    identifier: test_streamlit_deploy_snowcli
+    title: "My Fancy Streamlit"
+    stage: streamlit
+    query_warehouse: xsmall
+    main_file: streamlit_app.py
+    artifacts:
+      - streamlit_app.py
+      - pages/*.*
```

Comment:
Out of curiosity, and particular reason to use `*.*`, instead of just `*`? I've never seen that pattern used before.